AdvancedDLSupport.AOT/AdvancedDLSupport.AOT.csproj

```diff
@@ -28,12 +27,12 @@
     <Reference Include="System.Reflection.Emit" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="CommandLineParser" Version="2.2.1" />
-    <PackageReference Include="NLog" Version="4.5.3" />
-    <PackageReference Include="StrictEmit" Version="2.4.2" PrivateAssets="Compile;Build;Analyzers;ContentFiles"/>
+    <PackageReference Include="CommandLineParser" Version="2.5.0" />
+    <PackageReference Include="NLog" Version="4.6.5" />
+    <PackageReference Include="StrictEmit" Version="2.4.2" PrivateAssets="Compile;Build;Analyzers;ContentFiles" />
     <PackageReference Include="JetBrains.Annotations" Version="11.1.0" PrivateAssets="all" />
     <PackageReference Include="JetBrainsAnnotations.Fody" Version="2.2.0" PrivateAssets="all" />
-    <PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
+    <PackageReference Include="System.Reflection.Emit" Version="4.6.0-preview6.19303.8" />
```

Comment:
Is using a preview version a requirement? Would prefer not to.

Reply:
Ah this was me trying to figure out if corefx supported RunAndSave at all, will fix.

Reply:
Fixed.